frontend/src/components/VoteResultChart.css

```diff
@@ -35,6 +29,18 @@
   display: none;
```

Comment:
This is just details, but as the thumb and percentage are now wrapped in a single parent element, you can hide that as a whole. This fixes the case where both the thumb and the percentage are hidden, but you can still see the solid background:

<img width="805" alt="Screen Shot 2024-08-30 at 16 23 01" src="https://github.com/user-attachments/assets/575b0fc1-16b1-45a4-9895-9ce39c7fc80f">

There’s another instance of that at the end of the file for medium sized bars.

Reply:
Hah, good that you caught that. I actually did not see that :O :eyeglasses: 
Will fix and also see how to get the height to the intended value!